torch/quantization/fake_quantize.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from torch.nn import Module
+from .._jit_internal import weak_module
+from .observer import Observer, observer
+
+@weak_module
+class FakeQuantize(Module):
+    ''' Simulate the quantize and dequantize operations in training time.
+    Args:
+        `qconfig`: object that encodes configuration info for quantization
+        `observer_module`: Observer module that records stats of weights and
+        activations
+        `calcqparam`: A function that calculates quantization parameters
+        given the stats
+    '''
+
+    def __init__(self, qconfig, quant_min=-128, quant_max=127, quant_delay=0):
+        super(FakeQuantize, self).__init__()
+        self.observer = qconfig.activation()
```

Comment:
I think qconfig should just be the tuple (dtype, qscheme)
Why not just have: 
```
 def __init__(self, dtype = torch.qint8, qscheme = torch.per_tensor_symmetric,  quant_min=-128, quant_max=127, enable_fake_quant = True):
```
 Internally, fake quant can create an observer module
```